Tests/AppStoreConnectClientTests/AppStoreConnectClientTests.swift

```diff
@@ -389,4 +386,36 @@ final class AppStoreConnectClientTests: XCTestCase {
             XCTFail("Unexpected error: \(error)")
         }
     }
+    
+    func testFetchBuildsSuccessWithSortIsNil() async throws {
+        var mockClient = MockAPIClient()
+        mockClient.result = .ok
+        let client = AppStoreConnectClient(client: mockClient)
+        let app = Application(id: "Foo", bundleId: "Bar")
+        let expectedBuilds = [Build(schema: MockObjects.build)]
+        let build = try await client.fetchBuilds(
+            for: app,
+            with: BuildsQuery.init(
+                sort: nil,
+                fields: [.version, .minOsVersion, .uploadedDate]
+            )
+        )
+        XCTAssertEqual(build, expectedBuilds)
+    }
+    
+    func testFetchBuildsSuccessWithFieldsIsNil() async throws {
+        var mockClient = MockAPIClient()
+        mockClient.result = .ok
+        let client = AppStoreConnectClient(client: mockClient)
+        let app = Application(id: "Foo", bundleId: "Bar")
+        let expectedBuilds = [Build(schema: MockObjects.build)]
+        let build = try await client.fetchBuilds(
+            for: app,
+            with: BuildsQuery.init(
+                sort: [.hyphenVersion, .uploadedDate],
+                fields: nil
+            )
+        )
+        XCTAssertEqual(build, expectedBuilds)
+    }
```

Comment:
Need add test without query. SUT - `let build = try await client.fetchBuilds(for: app)`

Reply:
Fixed.